Cargo.toml

```diff
@@ -55,6 +55,7 @@ bitflags = "1.2.1"
 [build-dependencies]
 cargo_gn = "0.0.13"
 which = "3.1.0"
+regex = "1.3.1"
```

Comment:
Is it possible to accomplish this without adding the regex dependency? It seems like a large addition for a relatively simple task.

Reply:
I don't know. Parsing all these different `clang` version string seems to be quite a difficult task and the regex's from Node.js have been proven over time. You have to distinguish between Apple `clang` (which uses XCode version and had different version string formats over time) and llvm `clang` (where also distributions like to modify the format of the version string).